File: Backend/src/main/java/com/example/backend/Diagnovir.java
package com.example.backend;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class Diagnovir extends Order{

    private String result;

    public Diagnovir(String place, String time, String date, User owner, String result){
        super(place, date, time, owner,"Diagnovir");
        this.result = result;
    }
}

File: Frontend/src/classess/Diagnovir.ts
class Diagnovir extends Order {
  private _result: string;

  public constructor(place: string, time: string, date: string, owner: User, result: string) {
    super(place, date, time, owner, "Diagnovir");
    this._result = result;
  }

  public get result(): string {
    return this._result;
  }

  public set result(result: string) {
    this._result = result;
  }
}